composer.json

```diff
@@ -19,19 +19,14 @@
   },
   "require": {
     "composer/installers": "^1.3",
-    "concrete5/core": "^8.2",
+    "concrete5/core": "^9.0.0",
```

Comment:
Can you change this to `^9.0.1`? I'd also consider keeping ^8.2 but we should probably just go 9.0.1 minimum and let people change the requirement if they really need to.

Reply:
If you keep 8.2 as an option, will that work with `doctrine/dbal` and `concrete5/phpass-compat` requirements removed?